membership-common/src/main/scala/com/gu/stripe/StripeService.scala

```diff
@@ -58,7 +63,12 @@ class StripeService(apiConfig: StripeServiceConfig, client: FutureHttpClient)(im
   val httpClient: FutureHttpClient = client
   val publicKey: String = apiConfig.credentials.publicKey
   val paymentGateway: PaymentGateway = RegionalStripeGateways.getGatewayForCountry(apiConfig.stripeAccountCountry)
-  val paymentIntentsGateway: PaymentGateway = RegionalStripeGateways.getPaymentIntentsGatewayForCountry(apiConfig.stripeAccountCountry)
+  val paymentIntentsGateway: PaymentGateway =
```

Comment:
this whole StripeService class can be deleted as it's unused. I checked back ( https://github.com/guardian/members-data-api/pull/1069 ) to see if any tests were missed and there don't seem to have been any, so we can get rid of this one without losing any testing

Reply:
https://github.com/guardian/members-data-api/pull/1134